dist/tokens/native/themes/dark.d.ts

```diff
@@ -1,21 +1,25 @@
+export const shadow: {base : string};
+
 export const accent: { forest: { textBody: string, textEmphasis: string, backgroundMuted0: string, backgroundMuted1: string, backgroundMuted2: string }, sand: { textBody: string, textEmphasis: string, backgroundMuted0: string, backgroundMuted1: string, backgroundMuted2: string } };
-export const boxShadows : {cardBase : string,cardHover : string,popoverBase : string,modalBase : string};export const opacity : {overlayBackdrop : number};
\ No newline at end of file
+export const boxShadows: {cardBase : string,cardHover : string,popoverBase : string,modalBase : string};
+export const opacity: {overlayBackdrop : number};
+export const typography: { h1: { fontFamily: string ,fontWeight: number ,fontSize: string ,lineHeight: string ,textDecoration: string ,textCase: string  },h2: { fontFamily: string ,fontWeight: number ,fontSize: string ,lineHeight: string ,textDecoration: string ,textCase: string  },h3: { fontFamily: string ,fontWeight: number ,fontSize: string ,lineHeight: string ,textDecoration: string ,textCase: string  },h4: { fontFamily: string ,fontWeight: number ,fontSize: string ,lineHeight: string ,textDecoration: string ,textCase: string  },body: { fontFamily: string ,fontWeight: number ,fontSize: string ,lineHeight: string ,textDecoration: string ,textCase: string  },footnote: { fontFamily: string ,fontWeight: number ,fontSize: string ,lineHeight: string ,textDecoration: string ,textCase: string  },caption: { fontFamily: string ,fontWeight: number ,fontSize: string ,lineHeight: string ,textDecoration: string ,textCase: string  },subcaption: { fontFamily: string ,fontWeight: number ,fontSize: string ,lineHeight: string ,textDecoration: string ,textCase: string  },mobileH1: { fontFamily: string ,fontWeight: number ,fontSize: string ,lineHeight: string ,textDecoration: string ,textCase: string  },mobileH2: { fontFamily: string ,fontWeight: number ,fontSize: string ,lineHeight: string ,textDecoration: string ,textCase: string  },mobileH3: { fontFamily: string ,fontWeight: number ,fontSize: string ,lineHeight: string ,textDecoration: string ,textCase: string  },mobileH4: { fontFamily: string ,fontWeight: number ,fontSize: string ,lineHeight: string ,textDecoration: string ,textCase: string  },mobileBody: { fontFamily: string ,fontWeight: number ,fontSize: string ,lineHeight: string ,textDecoration: string ,textCase: string  },mobileFootnote: { fontFamily: string ,fontWeight: number ,fontSize: string ,lineHeight: string ,textDecoration: string ,textCase: string  },mobileCaption: { fontFamily: string ,fontWeight: number ,fontSize: string ,lineHeight: string ,textDecoration: string ,textCase: string  },mobileSubcaption: { fontFamily: string ,fontWeight: number ,fontSize: string ,lineHeight: string ,textDecoration: string ,textCase: string  }  };
```

Comment:
just sanity checking that we now want to access these tokens from here Theme? instead of previously from a `import { typography, border } from "aviary-tokens";`?

if so, we may want to consider moving `border` here so they are all from the same place